### PR TITLE
personal page

### PR DESCRIPTION
**Pull request by 0xAlison**

### CODE REVIEW

**Comment by 0xAlison** [comment]
basically after we fill in our personal information we'll be redirected to this page.
the pic uploading function has not been completed yet
Screen Shot 2019-07-08 at 1 30 00 AM

**Comment by nicolercl** [comment]
Hi Alison the page works fine, there's just a small typo for Self Intro~

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 1]
```diff
@@ -49,8 +51,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
...
+        jsonObject.addProperty("email", userData.getEmail());
+        jsonObject.addProperty("aboutMe", userData.getAboutMe());
+        jsonObject.addProperty("learnCate", userData.getLearnCategory());
+        jsonObject.addProperty("teachCate", userData.getTeachCategory());
+        jsonObject.addProperty("school", userData.getSchool());
+        jsonObject.addProperty("name", userData.getName());
+        jsonObject.addProperty("age", userData.getAge());
+        jsonObject.addProperty("skillLevel", userData.getSkillLevel());
+        //String json = gson.toJson(userData);
+        System.out.println(jsonObject);
```
Comment (nicolercl) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java:62]: I think this line is for debugging and can be removed.

File: src/main/webapp/user-page.html [hunk 2]
```diff
@@ -32,12 +36,42 @@
...
+                        <br>
+                        <h6 class="inline" >Skill Level: </h6><h6 id="level-container" class="inline">level not filled</h6>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
 
     <!--for aboutme box-->
+    <h3>Self Into</h3>
```
Comment (nicolercl) [inline src/main/webapp/user-page.html]: Small typo here.

**Review by nicolercl — APPROVED** [review]

**Pull request merged**